### ISSUE TITLE
ride.py started crashing without giving any errors

### ISSUE THREAD

**Issue by amittendulkar** [robotframework/RIDE#1876]
RIDE was working properly on my Windows 7 (64-bit) system with Python 2.7.14 until recently.

However, I am not sure what has changed, but it has started crashing at the launch time without giving any errors in the command line.

I am using the recommended installation of wxPython 2.8.12.1 (unicode) for Python 2.7.

In an attempt to resolve the issue, I upgraded both robotframework and robotframework-ride packages but still the UI is crashing,
```
Successfully installed Pypubsub-3.3.0 robotframework-ride-1.7.3.1 robotframeworklexer-1.1
```

Are there any places from where I can extract debug logs?

**Comment by amittendulkar** [comment]
Should I try `pip install wxPython`?

**Comment by amittendulkar** [comment]
Giving non-existent robot file as an argument while starting ride.py is not working either.

**Comment by HelioGuilherme66** [comment]
Please paste here all the output when starting ride.py from the command Window (including the prompt and command)

**Comment by amittendulkar** [comment]
Some interesting observations,

The below didn't work (nothing printed on command prompt),
```
C:\Users\amit.tendulkar>ride.py

C:\Users\amit.tendulkar>
```

Surprisingly, the below command worked. It opened RIDE with fresh profile (probably because I had removed AppData\Roaming\RobotFramework\ride\settings.cfg)
```
C:\Users\amit.tendulkar>python --version
Python 2.7.14

C:\Users\amit.tendulkar>python c:\Programs\Python27\Scripts\ride.py
```

After RIDE successfully started, I opened my root directory (through File -> Open Directory) containing robot files and MYLIB.py. I have kept my library's documentation file MYLIB.html in the same location. RIDE gave me the following warning,
```
Using test data in HTML format is deprecated. Convert 'C:
	\MYPATH\robotframework\lib\MYLIB.html' to plain text format.
```
Then everything worked and I was able to browse the test cases.

However, simple ride.py is still not working. Here is the command prompt output with 2 combinations.

```
C:\Users\amit.tendulkar>ride.py

C:\Users\amit.tendulkar>ride.py garbage.robot

C:\Users\amit.tendulkar>
```

For both of the above combinations, the UI got opened and I still got the MYLIB.html related error mentioned above. However, after closing the error window, the RIDE window also got closed without giving any errors in the command prompt.

Bottom line, I am unblocked with a workaround but RIDE might need to handle potential errors when run with `ride.py` command.

My ride.py has the below first (shebang) line
```
#!c:\programs\python27\python.exe
```

**Comment by HelioGuilherme66** [comment]
Alternative way to start RIDE.
```
python -m robotide.__init__
```
You can also try to create a desktop shortcut with:
```
python -m robotide.postinstall -install
```
Or if RIDE started, go to Tools>Create Desktop Shortcut.

**Comment by amittendulkar** [comment]
Thanks, the above tip (starting with `python -m robotide.__init__` and then creating shortcut through Tools) worked for me. Closing the issue.

**Comment by emmaisong** [comment]
Hi,
My "RIDE" robot framework quits each time I try to launch it. what could be the problem?

**Comment by emmaisong** [comment]
> Alternative way to start RIDE.
> 
> ```
> python -m robotide.__init__
> ```
> 
> You can also try to create a desktop shortcut with:
> 
> ```
> python -m robotide.postinstall -install
> ```
> 
> Or if RIDE started, go to Tools>Create Desktop Shortcut.

These did not help me at all, my RIDE still fails each time I launch
